scripts/relatedBills.py

```diff
@@ -36,38 +36,68 @@ def loadTitlesIndex(titleIndexPath=PATH_TO_TITLES_INDEX, zip=True):
 
     return titlesIndex
 
+def getSimilarTitles(titlesIndex: dict, same=True):
```

Comment:
Moved this function above the function (getRelatedBills) that calls it. Reorganized this to 1) create the billsRelatedByTitle dictionary from scratch in one pass. 2) Within the inner loop, get the index of the similar bill in the list, if it already exists. If it doesn't, add it to the list. If it does, append the current title to that bill's list of 'titles'. 2) add a 'same' Boolean parameter for when we want to match titles that are almost the same, but not exactly.